tests/test_gpu_examples.py

```diff
@@ -39,10 +40,13 @@
     WhisperTokenizer,
 )
 
+# A full testing suite that tests all the necessary features on GPU. The tests should
```

Comment:
Why move the comment here?

Reply:
Not sure why this was moved, it has been returned now